File: Day 18/solution.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        return accumulate(next(nums.begin()), nums.end(), 1, [it=next(nums.begin())](int acc, int &x) mutable {
            return (x != *(&x-1)) ? (*it++ = x, acc + 1) : acc;
        });
    }
};

File: Day 18/solution.py
Cách tiếp cận của mình:
- Vì mảng đã được sắp xếp nên chỉ cần 1 vòng lặp rồi so sánh giá trị hiện tại ở index với giá trị ngay sau đó.​
Nếu bằng nhau thì xóa thằng sau, tự động thằng kế tiếp sẽ thế chỗ -> so sánh tiếp, đến khi nào không bằng nhau nữa thì thằng giá trị hiện tại sẽ là duy nhất -> index + 1 để bắt đầu so sánh thằng tiếp theo và lặp lại những bước trên​
Python:
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        index = 0
        while index < len(nums)-1:
            if nums[index] == nums[index+1]:
                nums.remove(nums[index+1])
            else:
                index += 1  
        return len(nums)